src/main/java/uk/gov/ons/census/fwmt/jobservice/service/impl/JobServiceImpl.java

```diff
@@ -75,10 +92,26 @@ public void convertAndSendUpdate(UpdateFieldWorkerJobRequest updateRequest) thro
     gatewayEventManager.triggerEvent(String.valueOf(updateRequest.getCaseId()), COMET_UPDATE_SENT);
     if (!StringUtils.isEmpty(caseRequest.getPause())) {
       CasePauseRequest casePauseRequest = caseRequest.getPause();
-      cometRestClient.sendRequest(casePauseRequest, String.valueOf(updateRequest.getCaseId()));
+      ResponseEntity<Void> response = cometRestClient.sendRequest(casePauseRequest, String.valueOf(updateRequest.getCaseId()));
+      if (!isValidResponse(response)) {
+        String msg = "Unable to Create FieldWorkerJobRequest: HTTP_STATUS:" + response.getStatusCode() + ":" + response.getStatusCodeValue();
+        gatewayEventManager.triggerErrorEvent(this.getClass(), msg, String.valueOf(updateRequest.getCaseId()), FAILED_TO_CREATE_TM_JOB);
+        throw new GatewayException(GatewayException.Fault.SYSTEM_ERROR, msg);
+      }
+    }
+    ResponseEntity<Void> response = cometRestClient.sendRequest(caseRequest, String.valueOf(updateRequest.getCaseId()));
+    if (!isValidResponse(response)) {
+      String msg = "Unable to Create FieldWorkerJobRequest: HTTP_STATUS:" + response.getStatusCode() + ":" + response.getStatusCodeValue();
+      gatewayEventManager.triggerErrorEvent(this.getClass(), msg, String.valueOf(updateRequest.getCaseId()), FAILED_TO_CREATE_TM_JOB);
+      throw new GatewayException(GatewayException.Fault.SYSTEM_ERROR, msg);
```

Comment:
It looks like this entire if-block should be factored out into a function, as it is duplicated four times.